### DISCUSSION TITLE
Is it impossible to run GCNConv under GLIBC_2.23?

### DISCUSSION THREAD

**Discussion by songsong0425** [pyg-team/pytorch_geometric#6539] [Q&A] [answered]
Hi, I have a question about the `GCNConv` in `torch_geometric.nn`.
When I run the basic GCN code using PyG, there was OSerror like this:
```
import torch
import torch.nn.functional as F
from torch_geometric.nn import GCNConv

class GCNEncoder(torch.nn.Module):
    def __init__(self, feature_dim):
        super(GCNEncoder, self).__init__()
        self.conv1 = GCNConv(feature_dim, 256)
        self.conv2 = GCNConv(256, 16)

    def forward(self, data):
        x, edge_index = data.x, data.edge_index

        x = self.conv1(x, edge_index)
        x = F.relu(x)
        x = self.conv2(x, edge_index)

        return x
```
```OSError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by /usr/local/lib/python3.7/site-packages/torch_spline_conv/_basis_cuda.so)```

I checked that **my ubuntu version is 16.04 and GLIBC version is 2.23.**
Unfortunately, _I cannot change the ubuntu and GLIBC version_ because the server is not my own thing.
How can I run the GCNConv in this situation?

Thanks!

**Comment by rusty1s** [comment, marked as answer]
Try upgrading PyG or simply doing `pip uninstall torch-spline-cinv`.

**Reply by songsong0425** [reply]
Thank you for your reply! I'll ask to my server administrator as you suggest.
